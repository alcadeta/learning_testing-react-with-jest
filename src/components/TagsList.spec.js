import React from 'react';
import TagsList from './TagsList';
import renderer from 'react-test-renderer';

describe("The Tags List", () => {
    it("renders as expected", () => {
        const tree = renderer
            .create(<TagsList tags={["html", "css", "javascript"]} />)
            .toJSON();

        console.log(tree);

        expect(tree).toMatchSnapshot();
    });
});