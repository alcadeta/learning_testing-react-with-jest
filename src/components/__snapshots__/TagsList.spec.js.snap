// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Tags List renders as expected 1`] = `
<div>
  <code>
    html
  </code>
  <code>
    css
  </code>
  <code>
    javascript
  </code>
</div>
`;
